arion-compose.nix

```diff
@@ -1,14 +1,25 @@
-{ pkgs, ... }:
+{ pkgs, lib, config, ... }:
+
 let
+  inherit (config) share_dir nft-storage-key;
+  inherit (pkgs.lib) mkOption;
   # The address of the marketplace script (`MarketPlace.js` in `seabug-contracts`)
   marketplace-escrow-address = "addr_test1wr05mmuhd3nvyjan9u4a7c76gj756am40qg7vuz90vnkjzczfulda";
 
-  nft-marketplace-server = (import nft-marketplace-server/default.nix).packages.x86_64-linux."nft-marketplace-server:exe:nft-marketplace-server";
-  ogmios-datum-cache = (import ogmios-datum-cache/default.nix).packages.x86_64-linux."ogmios-datum-cache";
   # FIXME: CTL version also pinned in seabug-contract. We need only one source of truth
-  cardano-transaction-lib-server = (import
-    cardano-transaction-lib/default.nix).packages.x86_64-linux."ctl-server:exe:ctl-server";
-in {
+in
+
+{
+  options = {
```

Comment:
I couldn't find any documentation on how one uses these options, anyone know? For example how to change the value of `share_dir` when calling `arion up`

Reply:
Closest I found was this: https://nixos.org/manual/nixos/stable/index.html#sec-option-declarations, but it didn't really explain how to use the options for this case

Reply:
I've no much experience with Arion but it looks like a normal module, can't you e.g. add a `config.share_dir = "something";` in the `arion-compose.nix` file itself if you want to personalize that option?